js/PaperWallet.js

```diff
@@ -1,4 +1,40 @@
+import { newArray, newArray0, copyArray } from './utils/arrays';
```

Comment:
We don't have `newArray0` in the utils … i refactored it to a single `newArray` method that can be called with `true` as last param to indicate that it should nullify.

Reply:
newArray0 take an implicit size, not assume it from the input parameter. it doesn't work in this case. (one might say that newArray was miscalled in the first place and should be a copyArray like name)

Reply:
Ah ok… sorry then i misunderstood how it works … although the implementation looked 99% the same apart from this single param